libs/designsystem/src/lib/scss/base/_variables.scss

```
@@ -110,3 +110,21 @@ $item-heights: (
 /* Dropdown
 ****************************************************************************/
 $dropdown-item-height: map-get($item-heights, 's');
+
+/* Transitions
+****************************************************************************/
+$transition-duration: (
```

Align naming with existing maps:
```suggestion
$transition-durations: (
```

---

DOne